src/components/TodoFilter/TodoFilter.tsx

```diff
@@ -1,8 +1,24 @@
-export const TodoFilter = () => (
+import React from 'react';
+import { SortType } from '../../types/sortType';
+
+type Props = {
+  query: string;
+  setQuery: (value: string) => void;
+  sortType: SortType | string;
+  setSortType: (value: string) => void;
+};
+
+export const TodoFilter: React.FC<Props> = ({
+  query, setQuery, sortType, setSortType,
```

Comment:
A little more readable

```suggestion
  query,
  setQuery,
  sortType,
  setSortType,
```